interp/notation_ops.mli

```diff
@@ -63,10 +63,11 @@ exception No_match
 
 val print_parentheses : bool ref
 
-val match_notation_constr : bool -> 'a glob_constr_g -> interpretation ->
-      ('a glob_constr_g * extended_subscopes) list * ('a glob_constr_g list * extended_subscopes) list *
-      ('a cases_pattern_disjunction_g * extended_subscopes) list *
-      ('a extended_glob_local_binder_g list * extended_subscopes) list
+val match_notation_constr : bool -> 'a glob_constr_g -> Id.Set.t -> interpretation ->
```

Comment:
Can we make the set a labeled argument? Also, since we change the signature of this function, it would be nice to seize the opportunity of documenting it, at least with a short comment.

Reply:
I now used labels. There is a short documentation already, but 4 lines above.